source/codegen/metadata/nidcpower/functions.py

```diff
@@ -2500,6 +2541,32 @@
       }
     ],
     'returns': 'ViStatus'
+  },
+    'MeasureMultipleLcr': {
+    'cname': 'niDCPower_MeasureMultipleLCR',
```

Comment:
If you do end up wanting to implement this service method as part of this change you should add 'codegen_method': 'CustomCode' here and then define the implementation in `source\custom\nidcpower_service.custom.cpp`.

Reply:
Yes, I thought we could do it as part of the next user story and in subsequent PRs. In this PR, I am just adding metadata for it.